platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r33/GradleBuildModelCrossVersionSpec.groovy

```diff
@@ -76,7 +74,6 @@ class GradleBuildModelCrossVersionSpec extends ToolingApiSpecification {
         includedBuild.projects.size() == 4
     }
 
-    @TargetGradleVersion(">=3.1 <3.3")
     def "No included builds for old Gradle versions"() {
```

Comment:
❓ We should delete this test, since it is marked with `<3.3`, right?

Reply:
You are right, this should be deleted

Reply:
Right, this should be deleted.